3.5.13-SNAPSHOT fails to compile test due to mismatch of JDK versions

JmDNS make use of Mockito, specifically Mockito 5.2.0. However Mockito 5.x is supposed to run on JDK 11 or above.
When JmDNS is compiled with JDK 8 usage of Mockito fails due to this mismatch in JDK / class file versions:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:testCompile (default-testCompile) on project jmdns: Compilation failure
[ERROR] /Users/vtoni/git/jmdns/src/test/java/javax/jmdns/impl/DNSRecordTest.java:[19,19] cannot access org.mockito.Mockito
[ERROR]   bad class file: /Users/vtoni/.m2/repository/org/mockito/mockito-core/5.2.0/mockito-core-5.2.0.jar(org/mockito/Mockito.class)
[ERROR]     class file has wrong version 55.0, should be 52.0
[ERROR]     Please remove or make sure it appears in the correct subdirectory of the classpath.
[ERROR]
```

This issue could be solved by going back to Mockito 4.x (latest ATM is 4.11.0).

--- a/pom.xml
+++ b/pom.xml
@@ -52,6 +52,7 @@ The project was originally started in December 2002 by Arthur van Hoff at Strang
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
 		<jvm.version>1.8</jvm.version>
+		<mockito.version>4.11.0</mockito.version>
 	</properties>
 
 	<dependencies>
@@ -75,7 +76,7 @@ The project was originally started in December 2002 by Arthur van Hoff at Strang
 		<dependency>
 			<groupId>org.mockito</groupId>
 			<artifactId>mockito-inline</artifactId>
-			<version>5.2.0</version>
+			<version>${mockito.version}</version>
 			<scope>test</scope>
 		</dependency>
 	</dependencies>

